### DISCUSSION TITLE
Add editor for acct_gather.conf file

### DISCUSSION THREAD

**Discussion by NucciTheBoss** [canonical/hpc-team#6] [General]
The _acct_gather.conf_ file is used by Slurm to configure account gather plugins. An editor should be added for this configuration file because of the use case that Omnivector Solutions has. They use the InfluxDB plugin. Before they can adopt `slurmutils` in their workflow, they need this editor to be present.

Manual page: https://manpages.ubuntu.com/manpages/noble/en/man5/acct_gather.conf.5.html

**Comment by jamesbeedy** [comment]
Yes!

**Comment by NucciTheBoss** [comment]
Related PR here for adding the _acct_gather.conf_ editor here: https://github.com/charmed-hpc/slurmutils/pull/35

**Comment by NucciTheBoss** [comment]
`slurmutils` version 0.9.0 now includes an editor for the _acct_gather.conf_ file. Closing out this discussion since this work is now completed!
